components/Navigation.js

```diff
@@ -26,8 +26,11 @@ export default function Navigation({ pageStatus }) {
 const StyledGlobalNavigation = styled.nav`
   position: fixed;
   top: 0;
-  width: 375px;
+  width: 100%;
+  min-width: 375px;
+  max-width: 600px;
   background-color: var(--app-black);
+  z-index: 10;
```

Comment:
```suggestion
  z-index: 1;
```
usually one is enough :) maybe 2 if you use it somewhere else

Reply:
please make sure to delete commented out code (like below↓) before pushing :)

Reply:
Ichanged it to one and it works perfectly :)